packages/components/pie-button/src/index.ts

```diff
@@ -30,6 +30,8 @@ export class PieButton extends FormControlMixin(LitElement) implements ButtonPro
     connectedCallback () {
         super.connectedCallback();
 
+        this.setAttribute('data-test-id', `pie-button--${this.type}`);
```

Comment:
Does this need to live in the `connectedCallback`? Can it just be added to the render markup instead?

Reply:
I don't think we should be doing this at all. It prevents users from being able to assign their own test id to the button